src/Console/Shell.php

```diff
@@ -603,6 +603,7 @@ public function getOptionParser()
      *
      * @param string $name The task to get.
      * @return \Cake\Console\Shell Object of Task
+     * @throws \Exception
```

Comment:
Nothing here.

Reply:
load->_checkDuplicate RuntimeException
load->_throwMissingClassError Exception

Reply:
Not primary level, thus not really useful here, though.